internal/app/backend/backend.go

```diff
@@ -34,6 +75,14 @@ func BindService(p *appmain.Params, b *appmain.Bindings) error {
 	b.AddHandleFunc(func(s *grpc.Server) {
 		pb.RegisterBackendServiceServer(s, service)
 	}, pb.RegisterBackendServiceHandlerFromEndpoint)
-
+	if err := view.Register(
```

Comment:
This should be returning the error instead of logging fatal, and addding a closer
for unregistering.  Since this is a common pattern across BindService functions,
it can be generalized in appmain.

The BindService function doesn't really care about the view error either: It won't
add any context to it, and it should fail app startup.  So we can additionally
handle that by just setting on the side, and noticing it later.

```
// Bindings allows applications to bind various functions to the running servers.
type Bindings struct {
	sp *rpc.ServerParams
	a  *App
	firstErr error
}
```


```
func (b *Bindings) RegisterView(v *view.View) {
	err := view.Register(v)
	if err != nil {
	    if b.firstError == nil {
	      b.firstError = err
	    }
	    return
	}
	b.AddCloser(func() {
	  view.Unregister(v)
	})
}
```

in NewApplication
```
	err = bindService(p, b)
	if err != nil {
		surpressedErr := a.Stop() // Don't care about additional errors stopping.
		_ = surpressedErr
		return nil, err
	}

    if b.firstErr != nil {
		surpressedErr := a.Stop() // Don't care about additional errors stopping.
		_ = surpressedErr
		return nil, b.firstErr
    }
```